### PR TITLE
175 pf21 orwow vwget

### PR DESCRIPTION
**Pull request by kekuso**
- add tests and implementation for PF21 ORWOW VWGET

### CODE REVIEW

File: prototypes/fileparameters/pf21_orworVwget.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+const _ = require('lodash');
+const parameterService = require('../../../VDM/prototypes/parameterService');
+
+const RPC_EMULATOR_MODEL = [
+    {
+        id: 'ORWOR_VWGET',
+        name: 'ORWOR VWGET',
+        function: 'VWGET^ORWOR',
+        parameters: ['ORCH CONTEXT ORDERS'],
+        files: [''],
```
Comment (mfuroyama) [inline prototypes/fileparameters/pf21_orworVwget.js]: Teensy ask here: could you add `100.98` to the `files` array?  😁

Comment (kekuso) [inline prototypes/fileparameters/pf21_orworVwget.js]: can do, will do, done!

File: prototypes/rpcEmulatorUtils.js [hunk 2]
```diff
@@ -265,6 +265,73 @@ const invertFieldRange = (fileType, fieldId) => {
...
+        time = null;
+    } else return null;
+
+    // formatting for MM/DD/YYYY
+    if (format === 'MM/DD/YYYY') {
+        year = parseInt(date.substring(0, 3), 10) + 1700;
+        month = date.substring(3, 5);
+        day = date.substring(5, 7);
+        res = `${month}/${day}/${year}`;
+    }
```
Comment (mfuroyama) [inline prototypes/rpcEmulatorUtils.js]: This is cool! Eventually, if the need arises for more date/time format conversions, we might want to consider implementing this using a different strategy other than format-string comparison. For example, you might want to consider converting the Fileman date/time into a Moment.js object, then just ask Moment.js to format the date/time string for you. Just a thought...

Just something to think about tho, no need to implement that now.

Comment (kekuso) [inline prototypes/rpcEmulatorUtils.js]: Sounds good!

**Pull request merged**